pkg/controller/nodeconfig/config/ntp.go

```diff
@@ -439,3 +442,22 @@ func updateCondition(conditions []nodeconfigv1.ConfigStatus, c nodeconfigv1.Conf
 	}
 	return conditions
 }
+
+func reGenerateNTPConfig(ntpconfigs *nodeconfigv1.NTPConfig) *nodeconfigv1.NTPConfig {
+	if ntpconfigs.NTPServers == "" {
+		return ntpconfigs
+	}
+
+	// fileter the duplicated NTP servers
+	currentNTPServers := strings.Split(ntpconfigs.NTPServers, " ")
```

Comment:
just wondering why do we prefer " " over the comma as a splitter, is this a UI choice?

Reply:
Because the NTP servers on the config are split by a ` `, we need to convert it back to list.